src/monaco-utils.ts

```diff
@@ -53,4 +53,22 @@ export class MonacoUtils {
     MonacoUtils.TreeDefaults = TreeDefaults;
   }
 
+  static expandTree(tree: any) {
+    const model = tree.model;
+    const elements = [];
+
+    let item;
+    const nav = model.getNavigator();
+
+    while (item = nav.next()) {
+      elements.push(item);
+    }
+    if (elements.length != 0){
+      for (let i = 0, len = elements.length; i < len; i++) {
+        if (elements[i].element.name == "assembly" || elements[i].element.name == "src")
+        model.expand(elements[i]);
```

Comment:
this is very confusing formatting (same indentation level as `if`). Best to spell out `{` in most cases except one-liners.

Like
```
if (AUTO_EXPAND_DIRECTORIES.indexOf(elements[i].element.name) != -1) {
    model.expand(elements[i]);
}
```

example one-liner where it's reasonable to go without `{}`:

```
if (error) return done(error);
```

Reply:
changed

Reply:
where? Did you push?